data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/client/PeerForwarderClient.java

```diff
@@ -72,12 +69,12 @@ public AggregatedHttpResponse serializeRecordsAndSendHttpRequest(
 
```

Comment:
`ExecutorService` created in this class is not being used anymore, can you remove that?